examples/fluid/sphere_surface_tension_3d.jl

```diff
@@ -0,0 +1,30 @@
+# In this example we can observe that the `SurfaceTensionAkinci` surface tension model correctly leads to a
+# surface minimization of the water cube and approaches a sphere.
+using TrixiParticles
+using OrdinaryDiffEq
+
+fluid_density = 1000.0
+
+particle_spacing = 0.1
+cube_size = 0.9
+
+sound_speed = 20
```

Comment:
```suggestion
sound_speed = 20.0
```
Do we need this here? It's the same as in 2D.

Reply:
needed in the calculation of alpha

Reply:
Not resolved. Still an `Int`.